src/app.py

```diff
@@ -29,6 +31,12 @@ def get_prediction():
     data = request.get_json()
 
     input_text = data['query']
+    print(input_text)
+
+    label = "placeholder"
+
+    currentDT = datetime.datetime.now()
```

Comment:
in python wird (anders als in Java) kein CamelCase benutzt. Also statt currentDT -> current_dt